From: Stefan Hajnoczi <stefanha@gmail.com>
Date: Sun, 11 Nov 2012 13:52:25 +0100
Subject: [053/106] staging: line6: drop unused DEBUG_MESSAGES() macro

commit 60fb08b38eb1c9ffdde52817417f4b87f52be7b3 upstream.

The DEBUG_MESSAGES() macro is no longer needed since dev_dbg() is now
used for debug messages.

Signed-off-by: Stefan Hajnoczi <stefanha@gmail.com>
Signed-off-by: Greg Kroah-Hartman <gregkh@linuxfoundation.org>
---
 drivers/staging/line6/driver.h |    6 ------
 1 file changed, 6 deletions(-)

diff --git a/drivers/staging/line6/driver.h b/drivers/staging/line6/driver.h
index 35246cf..9dd8ff4 100644
--- a/drivers/staging/line6/driver.h
+++ b/drivers/staging/line6/driver.h
@@ -52,12 +52,6 @@
 
 #define LINE6_CHANNEL_MASK 0x0f
 
-#ifdef CONFIG_LINE6_USB_DEBUG
-#define DEBUG_MESSAGES(x) (x)
-#else
-#define DEBUG_MESSAGES(x)
-#endif
-
 #define MISSING_CASE	\
 	printk(KERN_ERR "line6usb driver bug: missing case in %s:%d\n", \
 		__FILE__, __LINE__)
